packages/core-data/src/entities.js

```diff
@@ -19,6 +19,9 @@ export const DEFAULT_ENTITY_KEY = 'id';
 
 const POST_RAW_ATTRIBUTES = [ 'title', 'excerpt', 'content' ];
 
+// Supports information is also available in the "edit" context of `/types`.
+const POST_TYPES_WITH_REVISIONS_SUPPORT = [ 'post', 'page' ];
```

Comment:
Not necessarily for now, but it would be good to consider moving the support information to the "view" context to avoid hardcoding this one.

Reply:
I'll add a note to test it. I'd also like to see if we can use the edit context, because we get all our post types info with it for free: https://developer.wordpress.org/rest-api/reference/post-types/#schema

But the response size is larger, so maybe a filter to add supports would suffice.

Reply:
No, we can't use the edit context probably because we want to load the entities for "contributors" as well who don't have access to the "edit" context.

Reply:
> we want to load the entities for "contributors" as well who don't have access to the "edit" context

Oh yeah 🤦🏻 I should have known. Thanks for the reminder. I have a follow up to investigate adding this info to the view context